libjsconsole/JSConsole.h

```diff
@@ -42,18 +44,35 @@ class JSConsole
 	void readExpression() const
 	{
 		std::string cmd = "";
-		g_logPost = [](std::string const& a, char const*) { std::cout << "\r           \r" << a << std::endl << std::flush; rl_forced_update_display(); };
+		g_logPost = [](std::string const& a, char const*)
+		{
+		    std::cout << "\r           \r" << a << std::endl << std::flush;
```

Comment:
according to [this stackoverflow answer](http://stackoverflow.com/questions/7013034/does-windows-carriage-return-r-n-consist-of-two-characters-or-one-character) it is